packages/agents/src/agents/ScriptWriter/utils.ts

```diff
@@ -0,0 +1,14 @@
+import { Chat, ContextWindow, Env, InMemoryWorkspace, LlmApi, Logger } from "@evo-ninja/agent-utils";
+import { ScriptWriter } from "./ScriptWriter";
+
+export const createScriptWriter = (args: {
+  llm: LlmApi,
+  chat: Chat,
+  logger: Logger,
+  env: Env
+}): ScriptWriter => {
+  const workspace = new InMemoryWorkspace();
+  const contextWindow = new ContextWindow(args.llm);
+  const chat = new Chat(args.chat.tokenizer, contextWindow, args.logger);
+  return new ScriptWriter(args.llm, chat, workspace, args.logger, args.env);
+};
```

Comment:
Is this factory really needed?

Reply:
This isn't new code, it already existed I just moved it into the proper directory (`ScriptWriter`).